### ISSUE TITLE
Not all transformed files are uploaded to Minio

### ISSUE THREAD

**Issue by ketan96-m** [ssl-hep/ServiceX#786]
**Problem**
Not all files are uploaded to Minio bucket.
Due to lack of resources for listing all the objects of the minio when objects are >1000, I used the CLI `mc` command to manually connect to a bucket when I submitted the transform to check the number of files uploaded. I found that not all files are uploaded to the bucket.

**To Reproduce**
1. pip install minio
2. https://min.io/docs/minio/linux/reference/minio-mc.html - downloaded the mc tool
3. Ran the bigger_uproot file
```
from servicex import ServiceXSpec, General, Sample
from servicex.func_adl.func_adl_dataset import FuncADLQuery
from servicex.servicex_client import deliver
import logging

logging.basicConfig(level=logging.INFO)
logger = logging.getLogger("servicex")
logger.setLevel(logging.DEBUG)


def bigger_uproot():
    query = FuncADLQuery().Select(lambda e: {'el_pt': e['el_pt']})

    spec = ServiceXSpec(
        General=General(
            ServiceX="servicex-uc-af",
            Codegen="uproot",
            OutputFormat="parquet",
            Delivery="SignedURLs"
        ),
        Sample=[
            Sample(
                Name="bigger_uproot",
                RucioDID="user.kchoi:user.kchoi.fcnc_tHq_ML.Wjets.v11", #1102 files
                Tree="nominal",
                Query=query
            )
        ]
    )
    print(len(deliver(spec)['bigger_uproot']))


if __name__ == "__main__":
    bigger_uproot()

```
4. `mc alias set ALIAS HOSTNAME ACCESS_KEY SECRET_KEY`
5. Added a breakpoint in the download files and allowed some time to pass to check if all files are uploaded
6. Count of files in bucket `mc find ALIAS/<BUCKET NAME> | find /c /v ""` (on WINDOWS so `find /c /v "" ` for counting lines)
7. Only gave back 1066 files

Bucket name: c7351070-70f8-4872-889c-655538c67193

**Expected behavior**
All files should be uploaded after transformation

**Comment by ponyisi** [comment]
HI @ketan96-m I think you need to tell us what the bucket name is ... otherwise we have to guess ... (and the bucket name is not a secret)

**Comment by ponyisi** [comment]
Believed fixed by #826
